src/test/regress/expected/ssl_by_default.out

```diff
@@ -0,0 +1,63 @@
+-- Citus uses ssl by default now. It does so by turning on ssl and if needed will generate
+-- self-signed certificates.
+-- To test this we will verify that SSL is set to ON for all machines, and we will make
+-- sure connections to workers use SSL by having it required in citus.conn_nodeinfo and
+-- lastly we will inspect the ssl state for connections to the workers
+-- ssl can only be enabled by default on installations of postgres 10 and above that are
+-- OpenSSL-enabled.
+SHOW server_version \gset
+SHOW ssl_ciphers \gset
+WITH features AS (
+    SELECT
+        substring(:'server_version', '\d+')::int >= 10 AS version_ten_or_above,
+        :'ssl_ciphers' != 'none' AS hasssl
+)
+SELECT (
```

Comment:
Do you think would it make sense to have another output file where Postgres is not compiled with SSL?

See a related issue: https://github.com/citusdata/tools/issues/39

Reply:
that output file was there, hence this query.

Given that both support for 9.6 and appveyor were dropped right before landing this diff it would be effectively dead code that would have been checked in, hence we removed it before landing.

Didn't know valgrind was build without ssl support, is there a specific reason for not having ssl in that build? I am asking because there are specifically some interesting things to test in this patch with valgrind, eg. if all openssl resources are correctly added to the memory context.

Otherwise feel free to add the output of the valgrind tests in a PR, happily approve.

Reply:
We also think that Valgrind tests should use SSL going forward and we should maybe re-run the valgrind tests for 8.1 with SSL enabled considering your note above.

I think the output file might be useful if any user/dev runs the tests without SSL support, not specifically for Valgrind tests.

I'll try to open a PR soon for that.